openc3-traefik/traefik.yaml

```diff
@@ -46,10 +46,10 @@ http:
       service: service-script
       priority: 7
     # Route to other tool plugins hosted statically in Minio
-    # Matches any path with a file extension which is assumed to be
-    # a static file
+    # Matches any path with a valid file extension which is assumed to be a static file
+    # TODO: We need to make all static tool files use a fixed route
     tools-router:
-      rule: Path(`/tools/{id:.*/.*[.].*}`)
+      rule: Path(`/tools/{id:.*/.*[.](ttf|otf|woff|woff2|html|js|css|png|jpg|jpeg|gif|svg|ico|json|xml|txt|pdf|zip|tar|gz|tgz|csv|tsv|md|yaml|yml|bin|doc|docx|xls|xlsx|ppt|pptx|mp4|mp3|wav|avi|mov|flv|swf|apk|ipa|deb|rpm|exe|msi|dmg|pkg|sh|bat|cmd|ps1|py|pl|rb|php|java|class|jar|war|ear|so|dll|lib|a|o|obj|pdb|pdb|lib|dylib|framework)}`)
```

Comment:
This is a hack. We should remove and force tools to route their static files from a known route like `/files`. We should also handle 404s better.